.yarnrc.yml

```diff
@@ -42,6 +27,22 @@ logFilters:
     level: discard
   - pattern: "wp-calypso@workspace:. provides react-dom (pf5aa8) with version 17.0.2, which doesn't satisfy what @types/wordpress__block-editor and some of its descendants request"
     level: discard
+  - pattern: "calypso@workspace:client provides react (pb0cb8) with version 17.0.2, which doesn't satisfy what @wordpress/block-editor and some of its descendants request"
```

Comment:
No clue why this is an issue now, but we can't get around it unless Gutenberg drops a transitive dependency.